locales/en.default.schema.json

```
@@ -146,6 +146,24 @@
         }
       }
     },
+    "search_input": {
+      "name": "Search input",
+      "settings": {
+        "header": {
+          "content": "Product suggestions"
+        },
+        "predictive_search_enabled": {
+          "label": "Enable product suggestions",
+          "info": "Affects the search bars in the header and on the search results page."
```

```suggestion
          "info": "For all search inputs."
```
Taking the conversation up to here @gretchen-willenberg, sorry for all the pings! 
This [is the setting reference](https://screenshot.click/21-08-kh9xm-jinb8.png). What do you think?

---

```suggestion
```

To remove finally and have info under the other settings. It's explicit that it would affect all search inputs.